web/package.json

```diff
@@ -48,5 +48,15 @@
   "devDependencies": {
     "http-proxy-middleware": "^2.0.2",
     "prettier": "^2.5.1"
+  },
+  "jest": {
+    "collectCoverageFrom": [
+      "src/**/*.{js,jsx}",
+      "!src/lib/cockpit.js",
+      "!src/reportWebVitals.js",
+      "!src/serviceWorker.js",
+      "!src/setupProxy.js",
+      "!<rootDir>/node_modules/"
```

Comment:
this line is not much clear for me ( even after I google jest rootDir ). Can you add comment with explanation?